src/components/icons/icons.tsx

```diff
@@ -0,0 +1,107 @@
+interface IIconProps {
+  Icon: React.FunctionComponent<React.SVGProps<SVGSVGElement>>;
+  width?: number;
+  height?: number;
+  className?: string;
+  color?: string;
+  onClick?: React.MouseEventHandler<SVGSVGElement>;
+}
+
+const IconSvg: React.FC<IIconProps> = ({
+  Icon,
+  width = 25,
+  height = 25,
+  color = '#265c79',
+  className,
+  onClick,
+}: IIconProps) => {
+  return (
+    <Icon
+      width={width}
+      height={height}
+      color={color}
+      className={className}
+      onClick={onClick}
+    />
+  );
+};
+
+export default IconSvg;
+
+// import { ReactComponent as HomeIconSvg } from '../styles/items/homeIcon.svg?react';
+// import { ReactComponent as MenuIconSvg } from '../styles/items/threelinesmenu.svg?react';
```

Comment:
Please remove commented out code since you don't need it and it just makes code not-so-clean.